backend/tests/test_api.py

```diff
@@ -1,4 +1,42 @@
+import pytest
+from backend.editor.api import api
+from falcon import testing
+
+
+@pytest.fixture(autouse=True)
+def test_setup(neo4j):
+    # delete all the nodes and relations in the database
+    query = "MATCH (n:p_test_branch:t_test:b_branch) DETACH DELETE n"
```

Comment:
Let's really delete all :-)

```suggestion
    # delete all the nodes and relations in the database
    query = "MATCH n DETACH DELETE n"
```